src/Hosting/test/testassets/BasicLinkedApp/BasicLinkedApp.csproj

```diff
@@ -5,6 +5,7 @@
     <OutputType>Exe</OutputType>
     <PublishTrimmed>true</PublishTrimmed>
     <TrimMode>link</TrimMode>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
How about just `<SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>`? So other warnings are still errors.

Reply:
I'll do that, but I do want to file issues for the warnings so I'd like them to show up.

Reply:
How about `<ILLinkTreatWarningsAsErrors>false</ILLinkTreatWarningsAsErrors>`

See https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming-options#suppressing-warnings